HPE Alletra storage driver (part 3) (#16445)
canonical/lxd: doc/api-extensions.md

Adds a new `bearer` authentication method and enables authentication to the DevLXD API.
See {ref}`DevLXD bearer tokens <devlxd-authentication-bearer>`.

## `storage_driver_alletra`

> Adds a new `alletra` storage driver which allows the consumption of storage volumes from a HPE Alletra Storage array.

Reviewer: ```suggestion
Adds a new `alletra` storage driver for the consumption of storage volumes from an HPE Alletra storage array.
```

> The following pool level configuration keys have been added:

Reviewer: ```suggestion
The following pool-level configuration keys have been added:
```
